### DISCUSSION TITLE
Problem with searching for symbols in Quick Switcher

### DISCUSSION THREAD

**Discussion by postnutcIarity** [BryanJenksCommunity/FAQ#134] [Q&A] [answered]
Hi,

I noticed that when Bryan searches for symbols like "(" for articles or "{" for books in Quick Switcher, it would just show the file names. Mine shows the folders before I see the file name which makes it quite difficult for me to see the list of articles I want to browse (Please see picture below). How do you guys remove the folder tree?

![image](https://user-images.githubusercontent.com/9300373/145733487-54b301a3-1fcf-41a6-b6aa-e0d54f136074.png)

**Comment by tallguyjenks** [comment, marked as answer]
it would just show the file names because all my files were at the root level of the vault, yours are in folders so that's why it appears that way

**Reply by postnutcIarity** [reply]
Thanks!
